testdata/dat.json

```diff
@@ -0,0 +1,40 @@
+[    {"ID": "2",
+    "Reading": "24",
+    "MinVol": "1",
+    "MaxVol": "200",
```

Comment:
Since the database is working now, there is no point in having this. Please remove it.